package.json

```diff
@@ -170,10 +169,10 @@
     "balena-config-json": "^2.1.1",
     "balena-device-init": "^5.0.2",
     "balena-device-status": "^3.2.1",
-    "balena-errors": "^4.2.1",
+    "balena-errors": "^4.3.0",
```

Comment:
Aligning with the dependency declared by `balena-sdk` (even though shrinkwrap actually installed v4.3.2 - it's OK).